### ISSUE TITLE
panic: kube2iam flag redefined: log_dir

### ISSUE THREAD

**Issue by keiths-osc** [jtblin/kube2iam#97]
With kube2iam 0.8.1, I am getting this:

```
kube2iam flag redefined: log_dir
panic: kube2iam flag redefined: log_dir

goroutine 1 [running]:
flag.(*FlagSet).Var(0xc42004a060, 0x1e7ce20, 0xc4201dd3d0, 0x156ce03, 0x7, 0x159238a, 0x2f)
	/usr/local/Cellar/go/1.9/libexec/src/flag/flag.go:793 +0x5e1
flag.(*FlagSet).StringVar(0xc42004a060, 0xc4201dd3d0, 0x156ce03, 0x7, 0x0, 0x0, 0x159238a, 0x2f)
	/usr/local/Cellar/go/1.9/libexec/src/flag/flag.go:696 +0x8b
flag.(*FlagSet).String(0xc42004a060, 0x156ce03, 0x7, 0x0, 0x0, 0x159238a, 0x2f, 0xc42054dea8)
	/usr/local/Cellar/go/1.9/libexec/src/flag/flag.go:709 +0x8b
flag.String(0x156ce03, 0x7, 0x0, 0x0, 0x159238a, 0x2f, 0x157f794)
	/usr/local/Cellar/go/1.9/libexec/src/flag/flag.go:716 +0x69
github.com/jtblin/kube2iam/vendor/github.com/golang/glog.init()
	/Users/jtblin/src/go/src/github.com/jtblin/kube2iam/vendor/github.com/golang/glog/glog_file.go:41 +0x14a
github.com/jtblin/kube2iam/vendor/k8s.io/apimachinery/pkg/util/runtime.init()
	<autogenerated>:1 +0x5f
github.com/jtblin/kube2iam/vendor/k8s.io/apimachinery/pkg/util/wait.init()
	<autogenerated>:1 +0x55
github.com/jtblin/kube2iam/k8s.init()
	<autogenerated>:1 +0x53
github.com/jtblin/kube2iam/server.init()
	<autogenerated>:1 +0x93
main.init()
	<autogenerated>:1 +0x5d
```

Version 0.7.0 seems to not have the issue.  It appears maybe [this previous issue](https://github.com/jtblin/kube2iam/issues/66) is back?

**Comment by jtblin** [comment]
Sorry about that, yes that was the same issue as previously when I ran `glide up`. Fixed that and republished `0.8.1`. `0.8.0` was not affected.

**Comment by jtblin** [comment]
Good point. Released `0.8.2` yesterday but forgot to mention that here.
